{
  "summary": {
    "outcome": "Failed",
    "testsRan": 1,
    "passing": 0,
    "failing": 1,
    "skipped": 0,
    "passRate": "0%",
    "failRate": "100%",
    "skipRate": "0%",
    "testStartTime": "Wed Aug 31 2022 13:54:45",
    "testExecutionTimeInMs": 295,
    "testTotalTimeInMs": 295,
    "commandTimeInMs": 78,
    "hostname": "https://cean-dev-ed.my.salesforce.com",
    "orgId": "00D4L000000FcKzUAK",
    "username": "tony@cean.com",
    "testRunId": "7074L00002dpeGI",
    "userId": "0054L000000oEXtQAM"
  },
  "tests": [
    {
      "id": "07M4L000000sQzFUAU",
      "queueItemId": "7094L000000HTtiQAG",
      "stackTrace": "Class.ceanff.CaseUtility.createCase: line 35, column 1\nClass.ceanff.ContactUtility.triggerAction: line 4, column 1\nClass.ceanff.TestContactUtility.executesTriggerAction: line 7, column 1",
      "message": "System.DmlException: Insert failed. First exception on row 0; first error: INVALID_CROSS_REFERENCE_KEY, Owner ID: owner cannot be blank: [OwnerId]",
      "asyncApexJobId": "7074L00002dpeGIQAY",
      "methodName": "executesTriggerAction",
      "outcome": "Fail",
      "apexLogId": "07L4L000004UsDLUA0",
      "apexClass": {
        "id": "01p4L000009obJGQAY",
        "name": "TestContactUtility",
        "namespacePrefix": "ceanff",
        "fullName": "ceanff__TestContactUtility"
      },
      "runTime": 295,
      "testTimestamp": "2022-08-31T12:54:46.000+0000",
      "fullName": "ceanff__TestContactUtility.executesTriggerAction",
      "diagnostic": {
        "exceptionMessage": "System.DmlException: Insert failed. First exception on row 0; first error: INVALID_CROSS_REFERENCE_KEY, Owner ID: owner cannot be blank: [OwnerId]",
        "exceptionStackTrace": "Class.ceanff.CaseUtility.createCase: line 35, column 1\nClass.ceanff.ContactUtility.triggerAction: line 4, column 1\nClass.ceanff.TestContactUtility.executesTriggerAction: line 7, column 1",
        "className": "ceanff",
        "compileProblem": "",
        "lineNumber": 35,
        "columnNumber": 1
      }
    }
  ]
}